backend/server.js

```diff
@@ -231,15 +261,20 @@ server.delete("/events/:id", isAuthenticated, async (req, res, next) => {
     try {
         let deletedEvent = await prisma.event.delete({
             where: { id: parseInt(eventId), organizer_id: sessionID },
-        }); 
+        });
         return res.json(deletedEvent);
     } catch (e) {
-        if (e.meta) { // Means it was a prisma error (no record found)
-            next({status: 404, message: "No event with the specified ID was found for this user"})
+        if (e.meta) {
```

Comment:
I'm curious, are these structural changes automated by a linter or something? I suggest making it a habit to run this once your PR commits are finalized, so they don't get grouped into future PRs!